frontend/src/components/Schedules/EditScheduleForm.tsx

```diff
@@ -58,61 +63,69 @@ export const EditScheduleForm: FC<Props> = ({ onUpdate, schedule }) => {
-          name={intl.get('description')}
-          defaultValue={schedule.description}
-          placeholder={intl.get('description')}
-          css=""
-        />
+        <div className={flexContainer}>
+          <Input
+            onChange={({ currentTarget: { value } }) => onUpdate(ScheduleKey.DESCRIPTION, value)}
+            name={intl.get('description')}
+            defaultValue={schedule.description}
+            placeholder={intl.get('description')}
+            css=""
+          />
+        </div>
       </div>
 
       <div className={flexWrapping}>
         <InlineFormLabel tooltip={intl.get('report_interval_description')}>
           {intl.get('report_interval')}
         </InlineFormLabel>
 
-        <Select
-          value={getIntervals().filter((interval: any) => interval.value === schedule?.interval)}
-          options={getIntervals()}
-          onChange={(selected: SelectableValue) => {
-            onUpdate(ScheduleKey.INTERVAL, selected.value);
-          }}
-        />
+        <div className={flexContainer}>
+          <Select
+            value={getIntervals().filter((interval: any) => interval.value === schedule?.interval)}
+            options={getIntervals()}
+            onChange={(selected: SelectableValue) => {
+              onUpdate(ScheduleKey.INTERVAL, selected.value);
+            }}
+          />
+        </div>
       </div>
 
       <div className={flexWrapping}>
         <InlineFormLabel tooltip={intl.get('report_group_description')}>{intl.get('report_group')}</InlineFormLabel>
 
-        <Select
-          value={reportGroups
-            ?.filter((reportGroup: ReportGroup) => reportGroup.id === schedule.reportGroupID)
-            .map((reportGroup: ReportGroup) => ({
+        <div className={flexContainer}>
+          <Select
+            value={reportGroups
+              ?.filter((reportGroup: ReportGroup) => reportGroup.id === schedule.reportGroupID)
+              .map((reportGroup: ReportGroup) => ({
+                label: reportGroup.name,
+                description: reportGroup.description,
+                value: reportGroup,
+              }))}
+            options={reportGroups?.map((reportGroup: ReportGroup) => ({
               label: reportGroup.name,
               description: reportGroup.description,
               value: reportGroup,
             }))}
-          options={reportGroups?.map((reportGroup: ReportGroup) => ({
-            label: reportGroup.name,
-            description: reportGroup.description,
-            value: reportGroup,
-          }))}
-          onChange={(selected: SelectableValue<ReportGroup>) => {
-            onUpdate(ScheduleKey.REPORT_GROUP_ID, selected?.value?.id ?? '');
-          }}
-        />
+            onChange={(selected: SelectableValue<ReportGroup>) => {
+              onUpdate(ScheduleKey.REPORT_GROUP_ID, selected?.value?.id ?? '');
+            }}
+          />
+        </div>
```

Comment:
Nit: just personal taste, but I prefer to have functions defined in the component body rather than inline... Also might be a justified memoisation for the  report group `map` + `filter` calculation (depending on how many report groups there are).